### ISSUE TITLE
Make Groupings for PDFs coming from the same domain.

### ISSUE THREAD

**Issue by dborg291** [alexweininger/recent-pdf#82]
Right now all online PDFs are listed in one giant list. It would make it easier for the user to find the PDF they are trying to find if each PDF was grouped based on the domain they where downloaded from. It would end up looking like a folder system. It would also be good for this to be a toggle so the user can decide which view they prefer.

Example:
-
If you have pdfs:
PDF1 from  google.com  
PDF2 from google.com  
PDF3 from moodle.com  
PDF4 from learding.com

It would end up looking like:

Google.com
PDF1
PDF2

PDF3
PDF4

**Comment by MagicCurlyHair** [comment]
Hi @dborg291 @alexweininger.
I can contribute and implement this. Please assign this issue to me and I will do it over the weekend.

**Comment by alexweininger** [comment]
@MagicCurlyHair I would like this to be a setting that the users can toggle.

**Comment by MagicCurlyHair** [comment]
@alexweininger Yeah, there's already a settings page implemented, so I will just add another option. I could do it as a simple boolean switch, but I think dropdown is better since more options for sorting could be implemented in the future (maybe users will be able to create and name folders themselves?)
Let me know what you think.

**Comment by alexweininger** [comment]
@MagicCurlyHair I think that's a great idea.

**Comment by MagicCurlyHair** [comment]
@alexweininger Cool, then I think it will be a 'File grouping' option with the following options:
* 'Ungrouped' - current view
* 'Group by domain' - the proposed grouping for domains.
Not sure about the naming though.

**Comment by alexweininger** [comment]
@MagicCurlyHair sounds great!
